felix/dataplane/linux/endpoint_mgr.go

```diff
@@ -1168,11 +1168,14 @@ func (m *endpointManager) configureInterface(name string) error {
 	acceptRAPath := fmt.Sprintf("/proc/sys/net/ipv6/conf/%s/accept_ra", name)
 	err := m.writeProcSys(acceptRAPath, "0")
 	if err != nil {
-		if exists, err := m.interfaceExistsInProcSys(name); err == nil && !exists {
+		if exists, err2 := m.interfaceExistsInProcSys(name); err2 == nil && !exists {
 			log.WithField("file", acceptRAPath).Debug(
 				"Failed to set accept_ra flag. Interface is missing in /proc/sys.")
 		} else {
-			log.WithField("ifaceName", name).Warnf("Could not set accept_ra: %v", err)
+			if err2 != nil {
+				log.WithError(err2).Error("Error checking if interface exists")
+			}
+			log.WithError(err).WithField("ifaceName", name).Warnf("Could not set accept_ra")
```

Comment:
Should we check if the accept_ra file (or if the whole ipv6 subtree) exists?  Just thinking if ipv6 is compiled out we'll log a warning.

Reply:
Perhaps we should, although I would hope that the error message that we'll now see has some sort of indication of that :)

I'm going to merge this for now because I think it's a strict improvement and was mostly a drive-by while investigating something else, though.